virttest/vt_cluster/__init__.py

```
@@ -0,0 +1,315 @@
+        try:
+            nodes_to_remove = list(partition.nodes)
+            for node in nodes_to_remove:
+                partition.del_node(node)
+
+            self._data["partitions"].remove(partition)
+            self._save()
+        except Exception as e:
+            LOG.error(f"Failed to remove partition {partition.uuid}: {e}")
+            raise ClusterError(f"Failed to remove partition: {e}")
+
+    def get_partition(self, conditions=None):
```

Ruff (0.13.1)

Unused method argument: `conditions`

(ARG002)